apps/ensadmin/src/components/indexing-status/view-models.ts

```diff
@@ -87,15 +103,19 @@ export function globalIndexingStatusViewModel(
 
 /**
  * View model for the network indexing status.
+ * @param chainId
  * @param chainName
  * @param networkStatus
  * @param firstBlockToIndexGloballyTimestamp
+ * @param chainsBlockExplorer - optional
```

Comment:
Let's make this one not optional and provide consistent UX for all chains 👍 
```suggestion
 * @param chainsBlockExplorer
```